tox.ini

```
@@ -23,12 +23,16 @@ commands =
 deps = black
 commands = black {posargs} arraylias test
 
-
 [testenv:docs]
+# Editable mode breaks macOS: https://github.com/sphinx-doc/sphinx/issues/10943
+usedevelop = False
 deps =
     -r{toxinidir}/requirements-dev.txt
 commands =
-  sphinx-build -b html -W {posargs} docs/ docs/_build/html
+  sphinx-build -j auto -b html -W --keep-going {posargs} docs/ docs/_build/html
```

FYI I recently discovered `-b html` is the default. It's fine to keep as before, but the command is a little bit easier to read with this change.

```suggestion
  sphinx-build -j auto -W --keep-going {posargs} docs/ docs/_build/html
```